rita_client/src/operator_update/mod.rs

```diff
@@ -169,6 +164,65 @@ pub async fn operator_update() {
         },
     });
 
+    // first check whats the last saved hr, send everything from that hr on
+    // but, we need to save a var of the last x hrs not seen, and load that up with the next checkin cycle if >0.
+    let current_hour = match get_current_hour() {
+        Ok(hour) => hour,
+        Err(e) => {
+            error!("System time is set earlier than unix epoch {:?}", e);
+            return;
+        }
+    };
+    let mut hours_to_send: u64 = 0;
+    let last = get_last_seen_hour();
+    if current_hour - last > 0 {
+        hours_to_send = current_hour - last;
+    }
+
+    let mut usage_data_client = get_usage_data(Client);
+    let mut usage_data_relay = get_usage_data(Relay);
+    let mut new_client_data: VecDeque<UsageHour> = VecDeque::new();
+    let mut new_relay_data: VecDeque<UsageHour> = VecDeque::new();
+    for _hour in 0..hours_to_send {
+        if new_client_data.is_empty() && new_relay_data.is_empty() {
+            break;
+        }
+        // Get Usage Data info from most recent save
+        let client_data = usage_data_client.pop_front();
+        let relay_data = usage_data_relay.pop_front();
+
+        // pop front, add to front of new vecdeque if exists.
+        match client_data {
+            Some(data) => {
+                let new_usage_hour = UsageHour {
+                    up: data.up,
+                    down: data.down,
```

Comment:
Changed logic to match this for simplicity, but with the note that UsageHour (rita common) and UsageHour (althea_types) are separately defined because of a cyclic dependency when adding these two to send through UsageHours in interop.rs with the rest of the ops data